This adds a library necessary for sparc.
This will probably not be sent upstream because it would be difficult to
add a sparc-only ifdef condition for this.

--- comm-esr31/mozilla/browser/installer/package-manifest.in.orig	2015-06-04 17:42:52.445166224 -0700
+++ comm-esr31/mozilla/browser/installer/package-manifest.in	2015-06-04 17:42:52.450333944 -0700
@@ -741,6 +741,7 @@
 bin/libfreebl_32fpu_3.so
 bin/libfreebl_32int_3.so
 bin/libfreebl_32int64_3.so
+bin/cpu/sparcv8plus/libnspr_flt4.so
 #endif
 
 ; [Updater]
